changelog/alias_from_trait.dd

```diff
@@ -0,0 +1,29 @@
+Aliases can be created directly from a `__trait`.
+
+Previously an `AliasSeq` was necessary in order to declare an alias based on a `__trait`.
+Now the grammar allows to write shorter declarations:
+
+---
+struct Foo
+{
+    int a;
+}
+
+alias oldWay = AliasSeq!(__traits(getMember, Foo, "a"))[0];
+alias newWay = __traits(getMember, Foo, "a");
+---
+
+To permit this it was more interesting to include `__trait` in the basic types
+instead of just changing the alias syntax, so as bonus,
+a new way of declaring the variables exists:
```

Comment:
I would reword this part. It's too specific to the compiler implementation.

Reply:
No it's not related to the implementation. This describes the language change.